VEDA_OS01/tests/test_views.py

```diff
@@ -221,3 +222,27 @@ def test_cielo24_error(self, mock_logger):
             json.dumps({'error': error_message}),
             status.HTTP_400_BAD_REQUEST
         )
+
+
+class HeartbeatTests(APITestCase):
+    """
+    Tests for hearbeat endpoint.
+    """
+    def test_heartbeat(self):
+        """
+        Test that heartbeat endpoint gives expected response upon success.
+        """
+        response = self.client.get(reverse('heartbeat'))
+        assert response.status_code == 200
+        assert json.loads(response.content) == {'OK': True}
```

Comment:
we can use `self.assertDict` here.

Reply:
For pytests, we should use assert statement. @mushtaqak your thoughts, you were discussing something like this yesterday?

Reply:
@muhammad-ammar If we are already inheriting from `APITestCase` anyway, why shouldn't we use `self.assertXXX..`?